### ISSUE TITLE
As training progresses, sampling time becomes longer and longer

### ISSUE THREAD

**Issue by destinyyzy** [YifeiZhou02/ArCHer#13]
As training progresses, sampling takes longer and longer. Have you encountered this problem during training? If so, what is the solution?

**Comment by YifeiZhou02** [comment]
Hi, thanks for your interest in our work. Which environment are you using? If it is Webshop, there is an known issue and its fix as in #6

**Comment by destinyyzy** [comment]
Yes, I encountered this problem in Webshop. I used the full amount of items and 10,000 instructions. I tried the method provided in #6, but it didn't help.

**Comment by YifeiZhou02** [comment]
I see, I also encountered this issue when I ran the experiments for the paper but was not able to solve it. This seems to be an issue related WebShop that processes html requests very slowly when there are a lot of instructions.
